docs/modules/servers/pages/distributed/configure/blobstore.adoc

```diff
@@ -35,6 +35,22 @@ Consequently, all the requested deletions will not be performed, meaning that bl
 
 NOTE: If you are upgrading from James 3.5 or older, the deduplication was enabled.
 
+=== Encryption choice
+
+Data can be optionally encrypted with a symmetric key using AES before being stored in the blobStore. As many user relies
+on third party for object storage, a compromised third party will not escalate to a data disclosure. Of course, a
+performance price have to be paid, as encryption takes resources.
+
+*encryption.aes.enable* : Optional boolean, defaults to false.
+
+If AES encryption is enabled, then the following properties MUST be present:
+
+ - *encryption.aes.password* : String
```

Comment:
it would be nice to provide sample commands to generate proper values for these 
something like: 
- generate password with : openssl rand -base64 64 
- generate salt with : openssl rand -hex 16 

note that I haven't checked if the provided sizes are actually sound or not ...